### ISSUE TITLE
Update to Current Features

### ISSUE THREAD

**Issue by acahir** [dhg/Skeleton#382]
As Skeleton seems to be unmaintained, I've forked Skeleton to use modern CSS features and kept it dependency free (ie. no SCSS required) so it's as simple as possible to get started:

[Barebones](https://github.com/acahir/Barebones)
- CSS Grid replaces 12 columns
- CSS Variables used globally for theme styling
- Maintained backwards legacy support for Skeleton styling but build on top of CSS Grid and CSS Variables

Check it out and hopefully others will find it useful.

**Comment by anonranger** [comment]
Thanks. I have been a long time user of getskeleton. Please continue to keep it updated.

**Comment by davidjendreizak** [comment]
@acahir Are you planning to keep this updated?

**Comment by acahir** [comment]
Yep, gonna try. Just had some minor changes submitted to Barebones, so at
least someone else found it useful.

**Comment by Mapiac** [comment]
@acahir did you implement smooth scroll and nav boilerplate yet curious. Tnx and super excited for the backwards compat w Skeleton.

**Comment by jamiebullock** [comment]
Thanks @acahir. Coming back to Skeleton after 4 years, it's great to see someone's taken up the maintenance.
